static/src/stylesheets/inline/article-photo-essay.scss

```diff
@@ -212,6 +212,14 @@
     }
 }
 
+.immersive-main-media__image-overlay {
```

Comment:
This might not be needed, but it is used here: https://github.com/guardian/frontend/blob/8721378c33106e4cd7df7ed4581ff9f117fb6903/common/app/views/fragments/photoEssay.scala.html#L57 for the photo essays.

Would the photo essays have a main media video? cc @blongden73